src/components/Anchor/index.js

```diff
@@ -2,10 +2,13 @@
 import { jsx } from '@emotion/core'
 import PropTypes from 'prop-types'
 import { styles } from './Anchor.styles'
+import { ReactComponent as Chevron } from 'static/icons/chevron.svg'
+import { Color } from 'lib/theme'
 
-export const Anchor = ({ children, ...rest }) => (
+export const Anchor = ({ children, withIcon, ...rest }) => (
```

Comment:
I know it isn't your work, but we're doing an `export const Anchor` here and then an `export default Anchor` below. Do you mind removing this named export in favor of the default export?